src/main/scala/sparkz/core/app/Application.scala

```diff
@@ -75,7 +75,7 @@ trait Application extends ScorexLogging {
     }
   }
 
-  val scorexContext = ScorexContext(
+  val scorexContext = SparkzContext(
```

Comment:
Can we rename it too as `sparkzContext`?

Reply:
Done, fixed all of these, thanks!